src/post_setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+sed --in-place '/^users:/ {N; s/users:.*/users: []/g}' /etc/cloud/cloud.cfg
```

Comment:
We're doing this because Dev was doing this. It's brought over from here:
https://github.com/cisagov/debian-packer/blob/e34570f730a04b680bcff5313384a1a13b4c935a/src/packer.pkr.hcl#L129

Reply:
Comments added in https://github.com/cisagov/ubuntu-server-packer/commit/f2efa0aa815856b7a5e40e6dc8e4e17884a9595c